website/blog/2024-08-15-webdriverio-v9-release.md

```diff
@@ -0,0 +1,194 @@
+---
+title: WebdriverIO v9 Released
+author: Christian Bromann
+authorURL: http://twitter.com/bromann
+authorImageURL: https://s.gravatar.com/avatar/d98b16d7c93d15865f34a225dd4b1254?s=80
+---
+
+The whole Webdriverio development team is stoked and proud to release WebdriverIO v9 today! 🎉 🎉 🎉
+
+This marks the begin of a new exciting era for all projects using WebdriverIO as their test automation tool. With the great work of browser teams that are working on e.g. Chrome and Firefox, we now enter a new era that offers much greater automation capabilities than ever before thanks to the new [WebDriver Bidi](https://w3c.github.io/webdriver-bidi/) protocol. With WebdriverIO v9 we have been working to be at the forefront of adopters of this new era and allow you to leverage from the power of the protocol first.
```

Comment:
begin -> beginning

_This marks the beginning of a new exciting era for all projects using WebdriverIO as their test automation tool_